test/testExplorer/TestExplorer.ts

```diff
@@ -10,17 +10,40 @@ const subscriptionId = urlSearchParams.get("subscriptionId") || process.env.SUBS
 const accountName = urlSearchParams.get("accountName") || "portal-sql-runner-west-us";
 const selfServeType = urlSearchParams.get("selfServeType") || "example";
 const iframeSrc = urlSearchParams.get("iframeSrc") || "explorer.html?platform=Portal&disablePortalInitCache";
-const token = urlSearchParams.get("token");
+const authToken = urlSearchParams.get("token");
 
-console.log("Resource Group:", resourceGroup);
-console.log("Subcription: ", subscriptionId);
-console.log("Account Name: ", accountName);
+const nosqlRbacToken = urlSearchParams.get("nosqlRbacToken") || process.env.NOSQL_TESTACCOUNT_TOKEN || "";
+const tableRbacToken = urlSearchParams.get("tableRbacToken") || process.env.TABLE_TESTACCOUNT_TOKEN || "";
+const gremlinRbacToken = urlSearchParams.get("gremlinRbacToken") || process.env.GREMLIN_TESTACCOUNT_TOKEN || "";
 
 const initTestExplorer = async (): Promise<void> => {
   updateUserContext({
-    authorizationToken: `bearer ${token}`,
+    authorizationToken: `bearer ${authToken}`,
   });
+
   const databaseAccount = await get(subscriptionId, resourceGroup, accountName);
+  const tags = databaseAccount?.tags;
+  const testAccountType = tags && tags["DataExplorer:TestAccountType"];
+
+  let rbacToken = "";
+  switch (testAccountType) {
```

Comment:
The way testAccountType is set up, it is expected that it could be undefined. Do you need a default case here?